custom_components/tahoma/cover_devices/vertical_cover.py

```diff
@@ -101,16 +106,37 @@ def current_cover_position(self):
     async def async_set_cover_position(self, **kwargs):
         """Move the cover to a specific position."""
         position = 100 - kwargs.get(ATTR_POSITION, 0)
-        await self.executor.async_execute_command(COMMAND_SET_CLOSURE, position)
+        if self.is_low_speed_enabled():
+            await self.executor.async_execute_command(
+                COMMAND_SET_CLOSURE_AND_LINEAR_SPEED, position, "lowspeed"
+            )
+        else:
+            await self.executor.async_execute_command(COMMAND_SET_CLOSURE, position)
 
     async def async_open_cover(self, **_):
         """Open the cover."""
-        await self.executor.async_execute_command(
-            self.executor.select_command(*COMMANDS_OPEN)
-        )
+        if self.is_low_speed_enabled():
+            await self.executor.async_execute_command(
+                COMMAND_SET_CLOSURE_AND_LINEAR_SPEED, 0, "lowspeed"
+            )
+        else:
+            await self.executor.async_execute_command(
+                self.executor.select_command(*COMMANDS_OPEN)
+            )
 
     async def async_close_cover(self, **_):
         """Close the cover."""
-        await self.executor.async_execute_command(
-            self.executor.select_command(*COMMANDS_CLOSE)
-        )
+        if self.is_low_speed_enabled():
```

Comment:
Why not check first if this is a low speed cover? Doesn't make sense to gather the entity on every call. (even for non low speed covers)

Reply:
I will update `is_low_speed_enabled` with a safeguard avoid the entity lookup.